### ISSUE TITLE
Fullscreen mode causes spike in CPU

### ISSUE THREAD

**Issue by getinstancemz** [olivierkes/manuskript#643]
When launching distraction free mode I noticed peformance degradation -- launched top to monitor and found CPU at approx 99%

![Screenshot_2019-09-14_11-59-57](https://user-images.githubusercontent.com/1153113/64907219-79288980-d6e7-11e9-8392-3b6d519abd8a.png)

This is on Fedora 27 / xfce. Reproduces reliably. I'm also running on a later Fedora.. I'll try there too.

Thanks!

**Comment by obw** [comment]
Which Version of Manusscript, I can't confirm this on Kubuntu 18LTS.

Manuskript 0.8.0 and Manuskript 0.9.0 develop

- Python 3.6.8
- PyQt 5.10.1
- Qt 5.9.5

The Max what I see is about 4% CPU for Manusscript, in a Scene with about 3000 Words

Over Menu -> Help -> About

You get a dialog where you can copy the three Versions (Python/QTPyLib/QT), also Manusscript Version.

**Comment by worstje** [comment]
Someone just beat me to the punch with most questions.

On my end, I cannot reproduce this either, but it makes sense given that I am using Windows.

Other questions worth asking:

1. What full screen theme are you using?
2. What spellchecker are you using? Assuming it is turned on, does it stop when you turn it off?
3. Is it the entire time that the full screen window is open, or is it only for a limited time when switching? And if so, is it only the first time you switch or every single time you switch?

**Comment by obw** [comment]
For my Setup, a little Note. I use dual-screen, One Monitor, in this case, Konsole running with HTOP and on the other, Manusskript in fullscreen.

(When I don't look at Openstreetmap, I'm writing a chase Scene in this moment... One which goes about 900KM through Germany ;) )

**Comment by getinstancemz** [comment]
Thanks both -- tl;dr - I think this was a transitory glitch at my end -- there is a 100% spike on both my fedora boxes on full screen start -- but I can no longer reproduce the long term effect I saw when I reported this.

Versions: 

Version 0.9.0
Software Versions in Use:
     Python 3.6.3
     PyQt 5.9.1
     Qt 5.9.6

Full screen themes:
I experienced this with 'writingdesk' and 'oldschool'

Spell check:
Not sure which spellchecker -- it's working off the peg within the app. Yes, it was enabled.. and yes  disabling seems to fix the issue -- a spike of CPU on launch and then it settles down quickly.

Period:
INITIALLY I saw this for an extended period -- starting at at launch of full screen mode and falling back when reverting mode. Now though, I only see it for 10 seconds or so when changing to full. Which makes this issue invalid.. sorry for the wasted time, thanks for looking into it though.
